src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,35 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/createDriver")
+public class CreateDriverController extends HttpServlet {
```

Comment:
```suggestion
public class AddDriverController extends HttpServlet {
```
maybe change this controller to begin with Add(as in checklist, or if u think it's ok, can leave like that)
- AddManufacturerController
-AddDriverController